docs/packages/margarita-components/index.md

```diff
@@ -3,6 +3,13 @@ title: Margarita Components
 sidebar_label: Margarita Components
 ---
 
-_This page is a stub._
+Margarita Components is a local package to this repository, which contains components that are specific and tailored to the margarita needs and business logic, by composing components from [@kiwicom/universal-components](https://kiwicom.github.io/margarita/docs/packages/universal-components/).
+
+examples of this components would be the `PassengerCard`, or the `CreditCardPaymentForm`. To launch the Storybook run
+
+````
+yarn storybook:platform:margarita-components
```

Comment:
And maybe we can use same form for this like in case of universal. https://kiwicom.github.io/margarita/docs/packages/universal-components/contributing#mobile

As I know myself, I'll probably run `yarn storybook:platform:margarita-components` first. And after it will fail I'll go back to documentation and read rest of the text about `platform` 😄 .
So it can be safer to directly mention both commands.

Reply:
Done!